srcalc/src/main/webapp/WEB-INF/views/fragments/referenceItem.jsp

```diff
@@ -0,0 +1,7 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+
+<ul class="patientNote">
+<c:forEach var="item" items="${displayItemParam.referenceInfo}">
```

Comment:
I like this List of Strings idea. It doesn't support anything we'd ever want to display, but it's simple and I think it will work for the scope of this project.